ci/coverage_publish.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
when you fill this in, reminder to put a warning about never setting `set -x` like we have in the other push files.